doc/GCI_MENTOR_GUIDELINES.md

```diff
@@ -36,9 +38,21 @@ lines like
 
 * A GCI issue needs to be put in the [staging list](https://github.com/publiclab/plots2/issues/3276) as well.
 
-* You also need to mention the category of the issue(Code, Design, Documentation/training, Quality Assurance, Outreach) and tell the instance count, i.e., how many people can take that issue in the issue description.
+* You also need to mention the category of the issue from these recurring tasks: Code, Design, Documentation/training, Quality Assurance, Outreach and add the instance count, i.e., how many people can take that issue in the issue description(https://github.com/publiclab/plots2/wiki/Google-Code-In-Tasks).
+
+* Labels which we can use at publiclab after creating issues are bug, design, enhancement, refactorization, documentation, help-wanted, and support.
 
 * For GCI issue you need to create the same task on the GCI dashboard as well.
 
 * You need to use extra-friendly tone at Public Lab with people.
 
+**PR Review Guidelines:**
+
+1. PR should use 'Fixes #issue_no" in PR body.
+
+2. Code should be modular.
+
+3. Tests must be written if needed. All tests must pass before approval.
+
+4. Suggest code changes if any. Else approve the PR by commentting in the PR.
```

Comment:
Please take a look on https://github.com/publiclab/plots2/issues/3563 and add the remaining stuff.

Reply:
Also take a look on https://docs.google.com/document/d/1-BesjjJnQGAi5A9sS8bJoepmxRSmGZvZ9-uffcbhG_k/edit and add remaining stuff.